components/relay/src/error.rs

```diff
@@ -0,0 +1,15 @@
+pub type Result<T> = std::result::Result<T, Error>;
+
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
```

Comment:
I note you have an "auth_token" in the constructor - will you need an error variant specific to this token being invalid (or maybe a more general "account issue")?

Reply:
Added `RelayApi` and `ApiErrorMessage` so consumers can handle server-side failures more gracefully.